docs/outputs.rst

```diff
@@ -150,4 +150,75 @@ I012   ignored            ign_add1
 
 Visual reports
 --------------
-We're working on it.
+Static visual reports can be generated by using the ``--png`` flag when calling
+tedana from the command line.
+Images are created and placed within the output directory, in a folder labeled
+``figures``.
+
+These reports consist of three main types of images.
+
+Component Images
+````````````
+.. image:: /_static/example_good_component.png
+  :align: center
+
+For each component identified by tedana, a single image will be created.
+Above is an example of an accepted component.
+These are designed for an up-close inspection of both the spatial and temporal
+aspects of the component, as well as ancillary information.
+
+The title of the plot provides information about variance, kappa and rho values
+as well as the reasons for rejection, if any (see above for codes).
+
+Below this is the component timeseries, color coded on the basis of its
+classification.
+Green for accepted, Red for rejected, Black for ignored or unclassified.
+
+Slices are then selected from sagittal, axial and coronal planes, to highlight
+the component pattern.
+By default these images used the red-blue colormap and are scaled to 10% of the
+max beta value.
+
+.. note::
+  You can select your own colormap to use by specifying its name when calling
+  tedana with ``--png-cmap``.
+  For example, to use the bone colormap, you would add ``--png-cmap bone``.
+
+Finally, the bottom of the image shows the Fast Fourier Transform of the
+component timeseries.
+
+Tip: Look for your fundamental task frequencies here!
+
+
+.. image:: /_static/example_bad_component.png
+  :align: center
+
+Above, you can appreciate a component that was rejected.
+In this case, the subject moved each time the task was performed - which
+affected single slice groups.
```

Comment:
I'm a little confused as to what "single slice groups" means! I think this could use a little more explanation (or not, maybe neuroscientists all understand this) 😄

Reply:
no, thats a bit confusing, I'll revise.

Reply:
I think I made it more clear - when someone moves in the scanner it effects single slices of the data (or can, atleast). But this data as more than one stripe, because multiple slices were collected at once. I don't know if slice groups is quite the right term, so I modified it to explain a bit more. 

What I would like to have in the end is more examples of artifacts - so folks can really appreciate what it is finding and (hopefully) removing from data. in time...